watertap/tools/oli_api/util/watertap_to_oli_helper_functions.py

```diff
@@ -70,15 +69,19 @@ def get_oli_name(watertap_name: str) -> str:
     :return oli_name: string name of a solute in OLI format
     """
 
-    components = watertap_name.split("_")
-    if len(components) == 0:
-        raise IOError(f" Unable to parse solute '{watertap_name}'.")
-    if len(components) == 1:
-        molecule = components[0]
-    elif len(components) == 2:
-        molecule = components[0] + "ION"
-    oli_name = molecule.replace("[", "").replace("]", "").upper()
-    return oli_name
+    exclude_items = ["temperature", "pressure", "volume"]
+    if watertap_name.lower() in exclude_items:
+        return watertap_name
```

Comment:
What made you add this? Just wondering what popped up that triggered a need for this since I figured one would only provide a solute name as input (imagining that you tried automatically looping through state vars or something like that but not sure).

Reply:
I added this because of how `build_survey` works: https://github.com/watertap-org/watertap/blob/7c0d684ee36ecee360513601942d54b289fa2bd3/watertap/tools/oli_api/flash.py#L100C5-L125C22

It does loop through each survey parameter and try to get an `oli_name` from it